Tasks/FoobarAzureV0/package-lock.json

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
This is an auto-generated file and will be created on build. needs to be checkedin.